app/backend/src/tests/test_communities.py

```diff
@@ -680,6 +680,14 @@ def test_ListDiscussions(testing_communities):
                 assert d.thread.thread_id > 0
                 assert d.thread.num_responses == 0
 
+    @staticmethod
+    def test_node_contained_user_ids_association_proxy(testing_communities):
+        with session_scope() as session:
+            c1_id = get_community_id(session, "Country 1")
+            node = session.query(Node).filter(Node.id == c1_id).one_or_none()
+            assert node.contained_user_ids == [1, 2, 3, 4, 5, 9, 10, 11]
```

Comment:
I don't understand where users 3, 4, and 5 come from home/how they got in there

Reply:
This seems right to me, "Country 1" has geom from 0 to 50, so any user with point value between those two should show up